samples/popup.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Popup Demo</title>
+    <meta name="description" content="Popup demo" />
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="chair.js"></script>
+    <script>
+      // TODO: Verify room is not needed and delete it
```

Comment:
Is this TODO part of this PR? With gravity gone, we may not need an actual room 😄

Reply:
yes, so i can remove that now i guess. I just wanted to confirm that